go.mod

```diff
@@ -2,6 +2,8 @@ module github.com/spiffe/spire
 
 go 1.19
 
+replace github.com/spiffe/go-spiffe/v2 => github.com/guilhermocc/go-spiffe/v2 v2.0.0-20230327182353-926e51ab6802
```

Comment:
The required change has been merged and released in version v2.1.4. Please update accordingly!